portfolio/src/main/java/com/google/sps/libraries/authstatus/AuthStatus.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.libraries.authstatus;
+
+/**
+* Class object to define the structure of a user comment.
```

Comment:
Nit: Is this comment still up-to-date? It seems to me that AuthStatus tells us whether a user has been authenticated?

Reply:
Yes! It was from the past class I made, thank you for pointing it out Aki!